### PR TITLE
support assign command

### PR DESCRIPTION
**Pull request by wq1019**
UCP: #6 

Signed-off-by: wq1019 <2013855675@qq.com>

### What problem does this PR solve? 
Support assign issue to contributor.

### What is changed and how it works?
Call GitHub API.

### Check List 

Tests 

 - No Test

### CODE REVIEW

**Comment by wq1019** [comment]
> @wq1019, you already got 200 points from easy level tasks when all pull requests merged. And you will not get score from this PR.

😒 👎

**Comment by you06** [comment]
@wq1019 Thanks for this PR, don't worry I'm fixing it. 😥

**Comment by you06** [comment]
> 
> 
> @wq1019, you already got 200 points from easy level tasks when all pull requests merged. And you will not get score from this PR.

No, you will get points from this PR, sorry for this accident.

File: pkg/providers/redeliver-command/event.go [hunk 1]
```diff
@@ -37,7 +37,8 @@ func (c *CommandRedeliver) ProcessIssueCommentEvent(event *github.IssueCommentEv
 	if !strings.Contains(comment, "/run") &&
 		!strings.Contains(comment, "/test") &&
 		!strings.Contains(comment, "/bench") &&
-		!strings.Contains(comment, "/release") {
+		!strings.Contains(comment, "/release") &&
+		!strings.Contains(comment, "/assign") {
```
Comment (you06) [inline pkg/providers/redeliver-command/event.go]: I think `/assign` does not need to be repeated.

Comment (wq1019) [reply pkg/providers/redeliver-command/event.go]: thanks, I'll fix it.

**Comment by you06** [comment]
> 
> 
> Congratulations, you get 300 score from easy level tasks in challenge program season-2, and if your PRs in reviewed stage all got merged, the score will be 300, try some medium and hard tasks!(you can not reward from easy and vector tasks now)

Oh I should remove the limitation in this repo.

**Pull request merged**